src/benchmark.rs

```diff
@@ -0,0 +1,43 @@
+mod structures;
+mod gtfs_structures;
+mod algo;
+extern crate chrono;
+extern crate cpuprofiler;
+extern crate itertools;
+#[macro_use]
+extern crate serde_derive;
+use chrono::prelude::*;
+use cpuprofiler::PROFILER;
+use itertools::Itertools;
+
+fn main() {
+    let gtfs = gtfs_structures::Gtfs::new("/home/tristram/workspace/csa/test_data/idf/").unwrap();
```

Comment:
It's strange it seems I cannot find this file when I run the bench :stuck_out_tongue_winking_eye:

Reply:
RTFM

Reply:
in the manual it's written that I must name my user `tristram` ?

Reply:
Yes, only Tristram’s are worthy.
I’ll fix the readme